Client/Frontend/ContentBlocker/ContentBlockerHelper.swift

```diff
@@ -254,27 +268,31 @@ extension ContentBlockerHelper {
         }
     }
 
-    fileprivate func compileListsNotInStore(completion: @escaping () -> Void) {
+    func compileListsNotInStore(completion: @escaping (_ success: Bool) -> Void) {
         guard let ruleStore = ruleStore else { return }
         let blocklists = blocklistBasic + blocklistStrict
-        let deferreds: [Deferred<Void>] = blocklists.map { filename in
-            let result = Deferred<Void>()
+        let deferreds: [Deferred<Error?>] = blocklists.map { filename in
+            let result = Deferred<Error?>()
             ruleStore.lookUpContentRuleList(forIdentifier: filename) { contentRuleList, error in
                 if contentRuleList != nil {
-                    result.fill()
+                    result.fill(nil)
                     return
                 }
                 self.loadJsonFromBundle(forResource: filename) { jsonString in
-                    ruleStore.compileContentRuleList(forIdentifier: filename, encodedContentRuleList: jsonString) { _, _ in
-                        result.fill()
+                    ruleStore.compileContentRuleList(forIdentifier: filename, encodedContentRuleList: jsonString) { _, error in
+                        if let error = error {
+                            print(error)
```

Comment:
remove the print

Reply:
Is there a preferred syntax for printing errors at the source of error?